src/components/SchedulePanel/components/CourseSchedule/index.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component, Fragment } from 'react';
+import Moment from 'moment';
+
+import Calendar from 'react-big-calendar/dist/react-big-calendar';
+import MomentLocalizer from 'react-big-calendar/lib/localizers/moment';
+
+import GordonLoader from '../../../../components/Loader';
+import schedule from './../../../../services/schedule';
+import myschedule from './../../../../services/myschedule';
+
+import './courseschedule.css';
+
+export default class CourseSchedule extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      loading: true,
+      officeHoursOpen: false,
+      disabled: true,
+      selectedEvent: null,
+      isDoubleClick: false,
+    };
+    this.eventInfo = [];
+  }
+
+  customEventPropGetter = (event, start, end, isSelected) => {
+    if (event.id > 1000) {
+      return {
+        className: 'custom-event',
+        style: {
+          backgroundColor: isSelected ? '#8d4987' : '#9b5094',
+        },
+      };
+    } else return {};
+  };
+
+  handleSelect = ({ start, end }) => {
+    const title = window.prompt('New Event name');
+    if (title)
+      this.setState({
+        events: [
+          ...this.state.events,
```

Comment:
I think this odd line is part of what you'll remove.

Reply:
Change made